string token
string timestamp
float64[] ego2global
int32[] labels
float64[] bboxes
float64[] scores